gui/frontend/src/search/search.coffee

```diff
@@ -665,14 +666,35 @@ angular.module "mindbender.search", [
             qsExtra = qsExtra || ''
             # TODO check if qsExtra is already there in @params.q
             qs = if (@getSourceFor @params.t)? then "q" else "s"
-            @params[qs] =
+            params_qs =
                 if newSearch or not @params[qs]
                     qsExtra
                 else if qsExtra and @params[qs].indexOf(qsExtra) == -1
                     "#{@params[qs]} #{qsExtra}"
                 else
                     @params[qs]
-            @doSearch no, no
+
+            # open in new tab?
+            if (event.shiftKey || event.ctrlKey || event.metaKey)
+                # clone params object
+                newParams = {}
+                for k,v of @params
+                    newParams[k] = v
+                # update params 
+                newParams[qs] = params_qs
+                if @params.ro == 'true'
+                    newParams['ro'] = 'false'
+                arr = []
+                for k,v of newParams
+                    if v != null
+                        arr.push k + '=' + encodeURIComponent(v)
+                url = $location.protocol() + '://' + location.host + '/#' + $location.path() + '?' + arr.join('&')
```

Comment:
Remove the protocol and host part. let the browser inherit it from current location.

Also, there should be some lib function to serialize the params, right? Say jQuery.param or something like that.